manageiq-providers-embedded_terraform.gemspec

```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "jwt", "~>2.2"
```

Comment:
NOTE we have some other gems which have outdated requirements for jwt locking it down to `jwt (~> 2.2.1)`
We should bump these since it is up to 2.8 now, so I wanted to leave this open on the minor to allow for that.

Reply:
https://github.com/IBM/ruby-sdk-core/pull/43

Reply:
I think this requires a Gemfile.lock.release change in backport - I'll make sure that happens

Reply:
I don't think so actually, we already have jwt in the Gemfile.lock.release
`3bf32eafaee (Jason Frey 2024-02-24 11:58:08 -0500  853)     jwt (2.2.3)`